Fix #520381 - add a Debian search to the default search list.
and #525991 - Adding debian bugs search to pre-installed searches
--- a/data/search
+++ b/data/search
@@ -19,3 +19,17 @@
 uri=http://www.thefreedictionary.com/%s
 icon=
 token=fd
+
+[Debian Packages]
+name=Debian Packages
+text=Search Debian Packages
+uri=http://packages.debian.org/%s
+icon=
+token=d
+
+[Debian Bugs]
+name=Debian Bugs
+text=Debian Bugs Search
+uri=http://bugs.debian.org/%s
+icon=
+token=dbs
